lib/backends/webgl/program-manager.ts

```diff
@@ -97,6 +101,7 @@ export class ProgramManager {
         } else {
           this.doDraw(buildArtifact, runData);
         }
+        gl.flush();
```

Comment:
we can remove the `gl.flush()` call inside `doDraw()`

Reply:
Thank you. that was left by mistake